### ISSUE TITLE
No images visible

### ISSUE THREAD

**Issue by opaque01** [a7digital/imagecycle#1]
On a TYPO3 7.6.16 installation imagecycle for tt_content are not visible.
```html
<div class="grey">
 <div id="c18" class="nopadding">
  <div class="csc-textpic csc-textpic-center csc-textpic-above csc-textpic-equalheight">
    <div class="csc-textpic-imagewrap tx-imagecycle-pi1" style="width:534px;">
      <ul id="imagecycle_18" class="tx-imagecycle-pi1-images" style="display:none;overflow:hidden;">
        <li class="csc-textpic-image csc-textpic-last"><a href="index.php?id=3&amp;L=0"><img src="fileadmin/_processed_/c/e/csm_c4b1405a0b.png" width="534" height="400" alt=""></a></li>
        <li class="csc-textpic-image csc-textpic-last"><a href="index.php?id=3&amp;L=0"><img src="fileadmin/_processed_/3/d/csm_0b7c6c66af.png" width="534" height="400" alt=""></a></li>
        <li class="csc-textpic-image csc-textpic-last"><a href="index.php?id=3&amp;L=0"><img src="fileadmin/_processed_/c/d/csm_28f92dd4d4.png" width="534" height="400" alt=""></a></li>
      </ul>
    </div>
  </div>
 </div>
</div>
```

**Comment by mbonaker** [comment]
Hi opaque01, there have been a lot of changes pushed to the repository today. Actually I am not sure whether any of them solves your problem but it might be worth a try to update to imagecycle 3.1.0

**Comment by karstenmadsen** [comment]
Following will fix the issue, if you are using pi1.

/Classes/Plugin/ImageCycle.php
`-require_once(ExtensionManagementUtility::extPath('imagecycle').'pi3/class.tx_imagecycle_pi1.php');`
`+require_once(ExtensionManagementUtility::extPath('imagecycle').'pi1/class.tx_imagecycle_pi1.php');`

/static/setup.txt
`-       userFunc = TYPO3Extension\Imagecycle\Plugin\NivoSlider->main`
`+    userFunc = TYPO3Extension\Imagecycle\Plugin\ImageCycle->main`

**Comment by opaque01** [comment]
The new version solved the problem! Thanks

**Comment by SicorDev** [comment]
Just to let you know, after downloading the master version today I still had to apply the changes suggested by karstenmadsen manually.

**Comment by mbonaker** [comment]
@SicorDev You are right, of course, the error was still there. I fixed it in e1cc341b09025830e307ae7b34f49b001c8a75b2 and 44a9b44e49b8692c33349c88b46e766ca68e61f5 so the new master (3.1.2) should have it solved. The suggestion from @karstenmadsen is included there.
